### PR TITLE
Update CI for v1

### PR DESCRIPTION
**Pull request by yangshun**
## Motivation

Per title.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

CI results.

## Related PRs

NA.

### CODE REVIEW

**Comment by endiliey** [comment]
Are the tests running ? @yangshun 

image

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,32 +66,62 @@ defaults: &defaults
...
       - checkout
-      - restore-cache: *restore-yarn-cache
-      - run: *yarn
-      - save-cache: *save-yarn-cache
+      - restore-cache: *root-restore-yarn-cache
+      - run: *root-yarn
+      - save-cache: *root-save-yarn-cache
       - run:
-          name: Check Prettier & ESLint
-          command: cd v1 && yarn ci-check
```
Comment (endiliey) [inline .circleci/config.yml]: `prettier` is no longer being run. Is this intentional ?

**Comment by yangshun** [comment]
Good catch @endiliey! I added a Prettier workflow which will run for both v1 and v2 code.

**Comment by endiliey** [comment]
I think we should run v2-tests as well. WDYT ?

If we want to be more efficient, then we can do some script to only run v1-test if v1 files is changed, and viceversa

https://github.com/facebook/jest/blob/master/.circleci/website.sh
```bash

**Comment by endiliey** [comment]
s

**Comment by yangshun** [comment]
I'll add v2 tests in a follow up PR. This PR is already too big :)

Regards,
Yangshun

**Pull request merged**